api/src/main/resources/workbench.yaml

```diff
@@ -111,6 +111,27 @@ paths:
           schema:
             $ref: "#/definitions/UsernameTakenResponse"
 
+  /api/v1/invitationKeyVerification:
+    post:
+      tags:
+        - profile
+      description: Verifies invitation key.
+      operationId: invitationKeyVerification
+      security: []
+      parameters:
+        - in: body
+          name: invitationVerRequest
+          schema:
+            $ref: "#/definitions/InvitationVerRequest"
+      responses:
+        "200":
+          description: Invitation Key verified.
+        "400":
+          description: Error occurred while verifying Invitation Key .
```

Comment:
Nit: spacing

Reply:
We can discuss about this tomorrow, as i did check and it has two spaces, i might be missing something.